lib/cmd/compile/media.js

```diff
@@ -22,7 +22,22 @@ function compile(options = {}) {
   const componentsSrc = afs.getComponents().map((comp) => ({ name: comp, path: path.join(afs.getComponentPath(comp), 'media', mediaGlobs) })),
     layoutsSrc = afs.getLayouts().map((layout) => ({ name: layout, path: path.join(process.cwd(), 'layouts', layout, 'media', mediaGlobs) })),
     styleguidesSrc = afs.getFolders(path.join(process.cwd(), 'styleguides')).map((styleguide) => ({ name: styleguide, path: path.join(process.cwd(), 'styleguides', styleguide, 'media', mediaGlobs) })),
-    sitesSrc = afs.getFolders(path.join(process.cwd(), 'sites')).map((site) => ({ name: site, path: path.join(process.cwd(), 'sites', site, 'media', mediaGlobs) }));
+    sitesSrc = afs.getFolders(path.join(process.cwd(), 'sites'))
+      .reduce((sites, site) => {
+        // adds multiple records for sites with subsites depending on existing media directories
+        // [ { name: site, path: sitePath }, { name: site/subsite, path: sitePath }, { name: site/subsite, path: subsitePath }]
+        // workflow for subsites: 1. copy parent into subsite directory, 2. copy subsite into subsite directory, overriding parent files
+        sites.push({ name: site, path: path.join(process.cwd(), 'sites', site, 'media', mediaGlobs) });
```

Comment:
We could have `process.cwd()` in a variable and reuse it every time we use it.

Reply:
created `cwd` const